content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Veneers"
+linkTitle: "Veneers (alpha)"
+weight: 5
+date: 2022-06-30
+---
+
+>Note: `veneers` are **ALPHA** functionality and may adopt breaking changes
+
+
+## Concept and Design
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions. FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/). However, FBCs can get complex, especially as the number of releases and dependencies scale. We introduce the concept of a `veneer` as an approach to simplifying interacting with FBCs.
+
```

Comment:
```suggestion

By using the `veneer` and the [opm](https://github.com/operator-framework/operator-registry) tool you can abstract the complexities to generate the content for your Operator package in the file-based-format.
```

Reply:
It's true that we currently support two veneer types, but a veneer is a simplification, not just the opm-supported one.  

That's why I broke out the veneer concept from the 'supported by opm' concept.  

If an author wants to use another approach which results in valid FBC, it would still be a valid veneer.

Reply:
@grokspawn can we try  something like 

 `veneer` can be supported and provided by other tools. In this document you will check how the `veneers` which are supported by [opm][opm-tool] tool can abstract the complexities to generate the content for your Operator package in the file-based-format.

Reply:
Maybe before `The veneers supported by `opm` are:`. would fit better

Reply:
Sure, I think we could just clarifies that a little bit more : https://github.com/operator-framework/olm-docs/pull/245#discussion_r927078092